templates/account.php

```diff
@@ -163,26 +163,55 @@
         }
     }
 
-    $is_child_license    = ( is_object( $license ) && FS_Plugin_License::is_valid_id( $license->parent_license_id ) );
-    $bundle_subscription = null;
+    $has_bundle_license              = ( is_object( $license ) && FS_Plugin_License::is_valid_id( $license->parent_license_id ) );
+    $bundle_subscription             = null;
+    $is_bundle_first_payment_pending = false;
 
     if (
         $show_plan_row &&
         is_object( $license ) &&
-        FS_Plugin_License::is_valid_id( $license->parent_license_id )
+        $has_bundle_license
     ) {
-        $bundle_subscription = $fs->_get_subscription( $license->parent_license_id );
+        $bundle_plan_title               = strtoupper( $license->parent_plan_title );
+        $bundle_subscription             = $fs->_get_subscription( $license->parent_license_id );
+        $is_bundle_first_payment_pending = $license->is_first_payment_pending();
     }
 
-    $is_active_bundle_subscription = ( is_object( $bundle_subscription ) && $bundle_subscription->is_active() );
-
     $fs_blog_id = ( is_multisite() && ! is_network_admin() ) ?
         get_current_blog_id() :
         0;
 
     $active_plugins_directories_map = Freemius::get_active_plugins_directories_map( $fs_blog_id );
 
     $is_premium = $fs->is_premium();
+
+    $account_addons       = $fs->get_updated_account_addons();
+    $installed_addons     = $fs->get_installed_addons();
+    $installed_addons_ids = array();
+
+    foreach ( $installed_addons as $fs_addon ) {
+        $installed_addons_ids[] = $fs_addon->get_id();
+
+        if (
+            ! $has_bundle_license &&
+            $show_plan_row &&
+            $fs_addon->has_active_valid_license()
+        ) {
+            $addon_license = $fs_addon->_get_license();
+
+            $has_bundle_license = FS_Plugin_License::is_valid_id( $addon_license->parent_license_id );

```

Comment:
@fajardoleo same as the comment above, semantically it should be:
```
if ( FS_Plugin_License::is_valid_id( $addon_license->parent_license_id ) ) {
    // Add-on's license associated with a parent/bundle license.
    $has_bundle_license = true;
}

Reply:
@fajardoleo also, the code in the `if` scope below can be merged with the logic of the proposed `if`.